munkipkg

```diff
@@ -639,6 +639,27 @@ def build(project_dir, options):
             if retcode:
                 raise BuildError("Could not remove existing %s" % outputname)
 
+        # remove .DS_Store files from the payload
+        retcode = subprocess.call(["/usr/bin/find",
+                                  build_info['payload'],
+                                  "-type", "f",
+                                  "-name", ".DS_Store",
+                                  "-exec", "rm", "-f", "{}", ";"])
+        if retcode:
+            print >> sys.stderr, (
+                "\nWARNING: Unable to remove .DS_Store files from payload.")
+
+        # make scripts executable
+        for pkgscript in ("preinstall", "postinstall"):
+            if os.path.exists(build_info['scripts'] + "/" + pkgscript):
+                retcode = subprocess.call([
```

Comment:
This should be done using Python's os.chmod instead of calling subprocess, and perhaps munki-pkg should tell us about changes its making so people can learn.